algebra/include/qdiff.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Phoenix-Pilot
+ *
+ * qdiff - quaternion-vector operations derivatives
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Mateusz Niewiadomski
+ *
+ * This file is part of Phoenix-Pilot software
+ *
+ * %LICENSE%
+ */
+
+#ifndef __PHOENIX_QVDIFF_H__
+#define __PHOENIX_QVDIFF_H__
```

Comment:
`__` prefix is reserved for stdlib functions
```suggestion
#ifndef PHOENIX_QVDIFF_H_
#define PHOENIX_QVDIFF_H_
```